app/controlplane/internal/biz/attestation.go

```diff
@@ -47,7 +47,7 @@ func NewAttestationUseCase(client CASClient, logger log.Logger) *AttestationUseC
 
 func (uc *AttestationUseCase) UploadToCAS(ctx context.Context, envelope *dsse.Envelope, backend *CASBackend, workflowRunID string) (*cr_v1.Hash, error) {
 	filename := fmt.Sprintf("attestation-%s.json", workflowRunID)
-	jsonContent, h, err := jsonEnvelopeWithDigest(envelope)
+	jsonContent, h, err := materials.JsonEnvelopeWithDigest(envelope)
```

Comment:
what about adding this helper in a new file under `internal/attestation/attestation.go`? It seems more related to `attestations` in the general sense than to materials. What do you think?